hash/dc.xml

```diff
@@ -429,7 +427,8 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 	-->
 
 	<software name="4wt">
-		<!-- http://redump.org/disc/18633/
+		<!-- http://redump.org/disc/18633/ -->
+		<!--
 		<rom name="4 Wheel Thunder (USA).gdi" size="1066" crc="3b5a27af" md5="be4f27fb70bb0b032699befd2dbccc9f" sha1="9cc0fb729d050ad59dd29547ba620e091677e008"/>
```

Comment:
The idea of having this verbosity over each entry doesn't really add value if you have to manually edit metadata in here.
Typical use scenario for QA:
1. search for supported flags;
2. search for a notes field if available;
3. 1 and 2 don't concile well both in reading and writing if they are far away in number of lines;

Consider moving the redump URL in a notes bucket field or a specific info node (`redump_url` ?), I'm also unsure about the value of having the tracks / cuesheet listed here when former does a much better job anyway.

Reply:
Those comments were already there, the only change was to move the redump source inside the software title so the comments stay with their titles.  Fully agree with you on the suggestions to convert this to notes/info tags but that wasn't what I was trying to tackle at this stage, but it could be a future clean-up item.